js/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "omi-summer-lab",
```

Comment:
Should it be named something to clarify this is just a JS SDK, not the whole OMI package?